src/pcm.c

```diff
@@ -1806,6 +1806,9 @@ int pcm_ioctl(struct pcm *pcm, int request, ...)
     arg = va_arg(ap, void *);
     va_end(ap);
 
-    // FIXME Does not handle plugins
+#ifdef TINYALSA_USES_PLUGINS
```

Comment:
This change is not needed as it is handled in https://github.com/tinyalsa/tinyalsa/commit/03f333e3b41f1a0e582689eed440efc6b0c1daea . Please remove this change from list.